### DISCUSSION TITLE
How to make errors multiline?

### DISCUSSION THREAD

**Discussion by mbozhilov-qb** [folke/trouble.nvim#380] [Q&A]
Hello peeps, 

I have the following issue. I'm getting some lengthy TS errors, but I can't really see what's happening because the lines get truncated. Any idea how to make errors be multiline?

<img width="1494" alt="image" src="https://github.com/folke/trouble.nvim/assets/107105763/0e453eb1-ccbb-47dd-8ad8-f3ac7529c8f2">
<img width="1417" alt="image" src="https://github.com/folke/trouble.nvim/assets/107105763/25ee4d24-755e-4b7c-a967-1fe38474f583">

**Comment by nik-rev** [comment]
I also have this issue. Did you find a solution?

**Reply by asynchroza** [reply]
> I also have this issue. Did you find a solution?

Not really, no. The only thing I figured is that you can navigate to the end of the line with some vim motions. Nevertheless, it's still hard to read.

**Reply by benomahony** [reply]
This is from core neovim. Try https://github.com/rachartier/tiny-inline-diagnostic.nvim/blob/main/lua/tiny-inline-diagnostic/diagnostic.lua though I needed to add a hack in my lazynvim options.lua to stop the lsp overriding the virtual text:

vim.diagnostic.config({ virtual_text = false })
vim.api.nvim_create_autocmd("LspAttach", {
  callback = function()
    vim.schedule(function()
      vim.diagnostic.config({ virtual_text = false })
    end)
  end,
})

I think the tiny plugin should be contributed back to core neovim, though I'd be happy to contribute it to lazy.nvim or snacks if that makes sense?

**Comment by anishkumar127** [comment]
I have using alternative way I'm using keymapping via showing in popup the error / diagnostic
